README.md

```diff
@@ -229,6 +230,23 @@ recorded values will not be recalculated if this value is changed. If you would
 like to change previous values, you would need to write a migration of the data
 to recalculate history.
 
+## Creating accounts
+
+By default, conbench has open read access, so a user account is not required to
+view results or read from the API. An account is required if the conbench instance
```

Comment:
Maybe "is required only if" (sorry apparently I can't suggest from the mobile interface!)